### PR TITLE
Scannable name fixes

### PR DESCRIPTION
**Pull request by penny64**
[QOL]
## About the PR 

Looking for mech scannable items that have identical names, so far we've got

wall phones "phone" -> "wall phone"
windooors "glass airlock" -> "thin glass airlock"

Research/Command/Engineering airlocks -> "department airlock"

Trying to find any of these that exist so please let me know if you know or find any

## Why's this needed? 

Manufacturers and rucks don't have icons at the moment, you can now scan items with the same name in #4989, and you can load items with the same name into manufacturers in #5033 , and this makes it possible to tell them apart in both

## Changelog

Nah

### CODE REVIEW

**Comment by Kubius** [comment]
I'd probably call the perspective thindoors something like "thin glass door" or "thin glass airlock", fwiw. They're mainly used over tables, but not exclusively.

**Comment by penny64** [comment]
Thanks a ton I was looking for a better name for that￼

File: code/obj/machinery/door/airlock.dm [hunk 1]
```diff
@@ -282,18 +285,22 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 	visible = 0
 
 /obj/machinery/door/airlock/glass/command
+		name = "command airlock"
 		icon = 'icons/obj/doors/Doorcom-glass.dmi'
 		req_access = list(access_heads)
 
 /obj/machinery/door/airlock/glass/engineering
+		name = "command airlock"
```
Comment (ZeWaka) [inline code/obj/machinery/door/airlock.dm:293]: 🤔

File: code/obj/machinery/door/airlock.dm [hunk 2]
```diff
@@ -282,18 +285,22 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
...
 		icon = 'icons/obj/doors/Dooreng-glass.dmi'
 		req_access = list(access_engineering)
 
 /obj/machinery/door/airlock/glass/medical
+		name = "medical airlock"
 		icon = 'icons/obj/doors/Doormed-glass.dmi'
 		req_access = list(access_medical)
 
 /obj/machinery/door/airlock/classic
+	name = "classic airlock"
```
Comment (ZeWaka) [inline code/obj/machinery/door/airlock.dm:303]: not a fan of 'classic'

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/machinery/door/airlock.dm (hunk 1 above)
Comment (penny64) [reply code/obj/machinery/door/airlock.dm:293]: I saw that an hour ago I haven't gotten to a computer!!!

File: code/obj/machinery/door/airlock.dm (hunk 2 above)
Comment (penny64) [reply code/obj/machinery/door/airlock.dm:303]: I'm trying "large" since that's the only natural way I can think to describe a non-perpective door

**Pull request merged**